cmd/sonobuoy/app/gen_test.go

```diff
@@ -85,19 +91,31 @@ func TestResolveConformanceImage(t *testing.T) {
 func TestResolveConfig(t *testing.T) {
 	defaultPluginSearchPath := config.New().PluginSearchPath
 	defaultAggr := plugin.AggregationConfig{TimeoutSeconds: 10800}
+	dynamicConfigFileName := "*determinedAtRuntime*"
```

Comment:
I use this to hot-swap out the name since I write the config to a temp file but want the cmdline input to be specified explicitly.